### PR TITLE
Directory with Filter

### PR DESCRIPTION
**Pull request by jvmontes**
- Add styled directory component with functional filter. 
- Add 3 orgs to mock directory data: ALPFA, Techqueria, and 1871. 
- Create Industry enum 

- Tested to make sure width of directory component doesn't change when items are filtered / unfiltered. 
- Tested to make sure height of directory doesn't decrease if there's less than 3 items displayed.
- Tested on mobile and desktop screens

Known issues:
- [ ] no empty state for directory. Ideally, we should have at least 1 option for each possible combination of filters. I added Cybersecurity and Real Estate as industries, but don't have Raices Cyber nor NAHREP added, which would apply to each industry. 
- [ ] Affinity enum isn't built out, but I wanted to place the idea. Something similar would be necessary for Locations (Chapters)
- [ ] Need unit tests to test Filter functionality with some mock data. 
- [x] ~~remove "X" when dropdown isn't open~~
- [x] ~~update styling to better reflect comps dropdown / filter button UI~~
- [x] ~~doesn't display selected industries anywhere when dropdown is closed~~
- [x] ~~DirectoryOrg displays industry_tag spans, ideally we turn these into nice little chips for enhanced UI.~~

### CODE REVIEW

File: src/app/ui/filter.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        } overflow-hidden`}
+      >
+        {industries.map((industry) => (
+          <label key={industry} className="flex items-center space-x-2 mb-2">
+            <input
+              type="checkbox"
+              checked={selectedIndustries.includes(industry)}
+              onChange={() => handleIndustryChange(industry)}
+            ></input>
+            <span>{industry}</span>
```
Comment (geraxrodriguez) [inline src/app/ui/filter.tsx:116]: I believe text is supposed to be black is currently showing up as white, same as background.

**Review by geraxrodriguez — CHANGES_REQUESTED** [review]
Really great work man. I learned a lot going through this. All I have for you is that text-color comment!

**Review by geraxrodriguez — APPROVED** [review]
Looks good!

File: src/app/ui/filter.tsx [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+            viewBox="0 0 40 32"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+            className="text-foreground"
+          >
+            <path
+              d="M2 2L38 5.11111M6 8.22222L34 11.3333M10 14.4444L30 17.5556M14 20.6667L26 23.7778M18 26.8889L22 30"
+              stroke="currentColor"
+              stroke-width="3"
+              stroke-linecap="round"
```
Comment (geraxrodriguez) [inline src/app/ui/filter.tsx:82]: Noticed an error in console: "Invalid DOM property stroke-width. Did you mean strokeWidth?".
Same thing for stroke-linecap. Changing to both to camelCase fixes error.

File: src/app/ui/filter.tsx [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+            width="14"
+            height="14"
+            viewBox="0 0 22 22"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              d="M2 2L11 11M20 20L11 11M11 11L20 2M11 11L2 20"
+              stroke="currentColor"
+              stroke-width="4"
```
Comment (geraxrodriguez) [inline src/app/ui/filter.tsx:104]: strokeWidth

File: src/app/ui/filter.tsx [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+            height="16"
+            viewBox="0 0 40 32"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+            className="text-foreground"
+          >
+            <path
+              d="M2 2L38 5.11111M6 8.22222L34 11.3333M10 14.4444L30 17.5556M14 20.6667L26 23.7778M18 26.8889L22 30"
+              stroke="currentColor"
+              stroke-width="3"
```
Comment (geraxrodriguez) [inline src/app/ui/filter.tsx:81]:
```suggestion
              strokeWidth="3"
```

File: src/app/ui/filter.tsx (hunk 2 above)
Comment (geraxrodriguez) [inline src/app/ui/filter.tsx:82]:
```suggestion
              strokeLinecap="round"
```

File: src/app/ui/filter.tsx (hunk 3 above)
Comment (geraxrodriguez) [inline src/app/ui/filter.tsx:104]:
```suggestion
              strokeWidth="4"
```

File: src/app/ui/directory.tsx [hunk 5]
```diff
@@ -1,44 +1,55 @@
...
-          <span className="px-3 py-1 bg-gray-200 rounded-full text-sm">
-            Tech
-          </span>
-          <span className="px-3 py-1 bg-gray-200 rounded-full text-sm">
-            Healthcare
-          </span>
-          <span className="px-3 py-1 bg-gray-200 rounded-full text-sm">
-            Finance
-          </span>
+    <section className="flex flex-col items-center justify-between pt-8">
```
Comment (geraxrodriguez) [inline src/app/ui/directory.tsx]: Looks like removing `justify-between` causes no change

File: src/app/ui/directory.tsx [hunk 6]
```diff
@@ -1,44 +1,55 @@
...
+      <h1 className="text-center pb-8 text-2xl">Directory</h1>
+      <div
+        className="w-10/12 md:w-3/4 shadow rounded-lg bg-background p-4 
+      min-w-[325px] min-h-[760px] md:min-h-[620px] md:min-w-[684px]
+      dark:shadow-gray-500"
+      >
```
Comment (geraxrodriguez) [inline src/app/ui/directory.tsx:25]: noticed that this <div> has a resizing, "jumping" effect that is more noticeable on screen sizes below 768px. 

When you use the filter for an industry that doesn't return any orgs (e.g. "cybersecurity"), the directory resizes down to it's minimum width. Jumps back if you select an industry that returns an org.

Not sure how big of an issue this is, but thought I would mention.

**Review by geraxrodriguez — CHANGES_REQUESTED** [review]
Left just a couple of comments for you. Nice work homie!

File: src/app/ui/directory.tsx (hunk 6 above)
Comment (jvmontes) [reply src/app/ui/directory.tsx:25]: @geraxrodriguez created a new bug and added to the backlog on Asana: https://app.asana.com/0/1207992233793749/1208379370797118/f

**Pull request merged**